### PR TITLE
docs: add an include statement for puppeteer-core in api.md

### PR DESCRIPTION
**Pull request by silent-lad**
I did this so as to resolve issue #3204 and to make doc better .

### CODE REVIEW

**Comment by silent-lad** [comment]
I signed it!

File: docs/api.md [hunk 1]
```diff
@@ -335,6 +335,12 @@ To sum up, the only differences between `puppeteer-core` and `puppeteer` are:
 
 In most cases, you'll be fine using the `puppeteer` package.
 
+But if you still need `puppeteer-core` remember to change the *include* line :-
```
Comment (aslushnikov) [inline docs/api.md]: let's move this below the "however" paragraph bith bullets: 

When using `puppetee-core`, remember to change the *include* line:
```js
const puppeteer = require('puppeteer-core');
```

Comment (silent-lad) [reply docs/api.md]: Doc changed as suggested by you . @aslushnikov

**Review by aslushnikov — APPROVED** [review]
Thanks!

**Pull request merged**